tests/WebAppCallsWebApiCallsGraph/TodoListService/Controllers/TodoListController.cs

```diff
@@ -16,9 +16,14 @@
 
 namespace TodoListService.Controllers
 {
-    [Authorize]
+    /* equivalent 
+    [Authorize(Policy = "RequiredScope(|AzureAd:Scope")]
+    [Authorize(Policy = "RequiredScope(User.Read")]
+    */
+    //[Authorize(Policy = "foo")]
     [Route("api/[controller]")]
-    [RequiredScope(RequiredScopesConfigurationKey = "AzureAd:Scopes")]
+    [RequiredScope("access_as_user")]
```

Comment:
We should discuss how we can mean this work in a cleaner way.

Reply:
I'll set something up for early next week. Thanks so much for all the great feedback.